tests/test_github.py

```diff
@@ -1767,14 +1775,12 @@ def test_github_enterprise_domain_fail(self, mocker: Any) -> None:
         mocker.patch('git_machete.cli.exit_script', mock_exit_script)
 
         github_enterprise_domain = 'git.example.org'
-        (
-            self.repo_sandbox.add_git_config_key('machete.github.domain', github_enterprise_domain)
-        )
+        self.repo_sandbox.add_git_config_key('machete.github.domain', github_enterprise_domain)
 
         expected_error_message = (
             "GitHub API returned `403` HTTP status with error message: `Forbidden`\n"
             "You might not have the required permissions for this repository.\n"
-            "Provide a GitHub API token with `repo` access via auth token for git.example.org from `~/.github-token`.\n"
+            "Provide a GitHub API token with `repo` access via None.\n"
```

Comment:
Huh `None`? 🤔

Reply:
@PawelLipski yes, it's better for consistency because it's independent of the developer settings and the previous value `auth token for git.example.org from ~/.github-token` was based on my personal token settings

Reply:
Huh okay... still, maybe instead of `None`, use some `"dummy"` string?

Reply:
Sure